src/Tests/Microsoft.Diagnostics.Monitoring.HostingStartup.UnitTests/ParameterCapturing/MethodSignatureTests.cs

```diff
@@ -0,0 +1,262 @@
+                        Type = null,
+                        Attributes = ParameterAttributes.None,
+                        IsByRef = false
+                    },
+                ]);
+
+        [Fact]
+        public void VarArgs() => TestCore(
+                typeof(StaticTestMethodSignatures).GetMethod(nameof(StaticTestMethodSignatures.VarArgs)),
+                "VarArgs",
+                "SampleMethods.StaticTestMethodSignatures",
+                typeof(StaticTestMethodSignatures).Module.Name,
+                [
+                    new (typeof(bool))
+                    {
+                        Name = "b",
+                        Type = "System.Boolean",
+                        Attributes = ParameterAttributes.None,
+                        IsByRef = false
+                    },
+                    new (typeof(int[]))
+                    {
+                        Name = "myInts",
+                        Type = "System.Int32[]",
+                        Attributes = ParameterAttributes.None,
+                        IsByRef = false
+                    },
+                ]);
+
+        [Fact]
+        public void UnicodeParameters() => TestCore(
+                typeof(StaticTestMethodSignatures).GetMethod(nameof(StaticTestMethodSignatures.Unicode_ΦΨ)),
+                "Unicode_ΦΨ",
+                "SampleMethods.StaticTestMethodSignatures",
+                typeof(StaticTestMethodSignatures).Module.Name,
+                [
+                    new (typeof(bool))
+                    {
+                        Name = "δ",
+                        Type = "System.Boolean",
+                        Attributes = ParameterAttributes.None,
+                        IsByRef = false
+                    },
+                ]);
+
+        [Fact]
+        public void NestedStructs() => TestCore(
+                typeof(StaticTestMethodSignatures.SampleNestedStruct).GetMethod(nameof(StaticTestMethodSignatures.SampleNestedStruct.DoWork)),
+                "DoWork",
+                "SampleMethods.StaticTestMethodSignatures+SampleNestedStruct",
+                typeof(StaticTestMethodSignatures).Module.Name,
+                [
+                    new(typeof(StaticTestMethodSignatures.SampleNestedStruct))
+                    {
+                        Name = "this",
+                        Type = "SampleMethods.StaticTestMethodSignatures+SampleNestedStruct",
+                        Attributes = ParameterAttributes.None,
+                        IsByRef = false
+                    },
+                    new(typeof(int))
+                    {
+                        Name = "i",
+                        Type = "System.Int32",
+                        Attributes = ParameterAttributes.None,
+                        IsByRef = false
+                    },
+                ]);
+
+        private static void TestCore(
+                                                                    MethodInfo methodInfo,
```

Comment:
At least in the GitHub view, the indentation seems off here - might just be me

Reply:
I think it's Github, it looks ok on my computer. Having said that, I did update the formatting a bit by moving the method call on a new line.